insights/core/__init__.py

```diff
@@ -1161,6 +1181,29 @@ class parser :py:class:`TextFileOutput`.
     * A None value when there is no timestamp info in the log file
     """
 
+    def _handle_content(self, context):
```

Comment:
Question: From the implementation, it looks like that `LogFileOutput` only supports lazy content loading, is this correct?

Reply:
Yup, I don't think it's required to add another one base class for this feature, as it doesn't affect the rules' execution on insights-archvies.